src/Middleware/UserPageDecorationMiddleware.php

```diff
@@ -135,8 +135,9 @@ function ($a, $b) {
 
         $tpl_data['subtest'] = $request->getAttribute("pageclass")->page ?? null;
 
-        $page = $request->getAttribute("pageclass");
-        if (method_exists($page, 'getFeedbackPanel')
+        $page = $request->getAttribute("pageclass") ?? '';
```

Comment:
There's no reason to use `?? ''` here, $page is supposed to be a NDB_Page, not a string, and null is a more accurate value. You should just use `$page !== null` instead of `!empty($page)` in the if statements.